tests/vectordb/test_elasticsearch.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+from unittest.mock import patch
+
+from embedchain.vectordb.elasticsearch import ElasticsearchDB
+
+
+class ElasticsearchDBTest(unittest.TestCase):

```

Comment:
Can you please use pytest instead of unittest? We are moving away from unit tests and will be using `pytest` instead.

Reply:
Ok, I will work on a pytest

Reply:
@Clement-Okolo any updates on the tests? Thanks!